tests/project_tests.ts

```diff
@@ -11,4 +11,7 @@ export const test = () => {
   })
 }
 
-test()
+test().catch((e) => {
```

Comment:
dumb nit but I don't love .catch() mainly because people might thing it's the proper thing to do in other cases (when in fact you should use try/catch). We could extract this into a function... or just bump TS to ES2017 so we can use top level async/await :)

Reply:
I think we already target ESNEXT, the issue is that we have a module:commonjs which doesn't have top level await.
Can switch them to immediately invoked ones, sure.